widget.json

```diff
@@ -84,6 +84,7 @@
       "js/build-lists.js",
       "js/build.js",
       "css/build.css",
+      "vendor/moment-with-locales.js",
```

Comment:
This shouldn't be necessary. The `moment` dependency should come with all the locale information already.